<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dossier medicale</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
border-bottom-color: rgb(11, 11, 11);
font: 9pt &quot;PT Bold Stars&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Gestion Dossier medicale</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1211</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter un Dossier Medicale</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>531</width>
          <height>361</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajout</string>
        </property>
        <widget class="QPushButton" name="anuuler_ajout">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>310</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>240</y>
           <width>251</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID Patient</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="le_date_ajout">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>160</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de creation</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Num dossier</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_num">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>80</y>
           <width>251</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="consulter">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>310</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Consulter</string>
         </property>
        </widget>
       </widget>
       <widget class="QTableView" name="tabdos1">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>150</y>
          <width>591</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher les Dossier Medicales</string>
       </attribute>
       <widget class="QTableView" name="tabdossier">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>881</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Patient</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rech_doc">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>20</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>RECHERCHE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="trie_doc">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>70</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIEE PAR NUM</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Supprimer un Dossier Medicale</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>551</width>
          <height>251</height>
         </rect>
        </property>
        <property name="title">
         <string>Suppression</string>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>60</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero de dossier </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_num_sup">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>110</y>
           <width>201</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>180</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Modifier un Dossier medicale</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>801</width>
          <height>371</height>
         </rect>
        </property>
        <property name="title">
         <string>Modification</string>
        </property>
        <widget class="QPushButton" name="annuler_modif">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>300</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>300</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID Patient</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_idpatient_modif">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>230</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>date de creation</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="le_date_modif">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>140</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_num_modif">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>70</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Num dossier</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Imprimer dossier medicale</string>
       </attribute>
       <widget class="QPushButton" name="Impdos">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>50</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="imprimer">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>781</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Gestion de Consultation</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1211</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Ajouter ou Modifier consultation</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>571</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Détail Dossier</string>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Num Dossier</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>40</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>N°intervention</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="numc">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nintervaj">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>40</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>571</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Proprieté</string>
        </property>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date inter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>40</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Problemme</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dinteraj">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>40</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="probaj">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>40</y>
           <width>271</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>571</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Suivi</string>
        </property>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tension</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Glycémie</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>40</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Poids</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>100</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Taille</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="tnaj">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="glaj">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>100</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="pdaj">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>40</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="tlaj">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>100</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>20</y>
          <width>471</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>Ordonnance</string>
        </property>
        <widget class="QLineEdit" name="ordaj">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>40</y>
           <width>461</width>
           <height>121</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="ajouterconsu">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Quitterc">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modif">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Afficher les Dossier Medicales</string>
       </attribute>
       <widget class="QTableView" name="tabconsu">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>981</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="intervcherch">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>N°intervention</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rechc">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>20</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>RECHERCHE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="triec">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>70</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIEE PAR NUM</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Supprimer un Dossier Medicale</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>551</width>
          <height>251</height>
         </rect>
        </property>
        <property name="title">
         <string>Suppression</string>
        </property>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>50</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero d intervention</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="numinsup">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>110</y>
           <width>201</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="supc">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>180</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Imprimer dossier medicale</string>
       </attribute>
       <widget class="QTextEdit" name="impc">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>921</width>
          <height>491</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Impdos_2">
        <property name="geometry">
         <rect>
          <x>1000</x>
          <y>50</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
